src/main/java/net/shadowmage/ancientwarfare/automation/gui/GuiWarehouseControl.java

```diff
@@ -105,6 +106,7 @@ public void setupElements() {
 		area.clearElements();
 		addInventoryViewElements();
 		storedLabel.setText(I18n.format("guistrings.warehouse.storage", getContainer().currentStored, getContainer().maxStorage));
+		storedLabel.setColor((getContainer().currentStored == getContainer().maxStorage) ? Color.RED.getRGB() : 4210752); // red if full, gray otherwise

```

Comment:
so I am a bit of an extremist and really don't like using comments for what code could tell us. so this comment shouldn't be here and perhaps the condition of `getContainer().currentStored == getContainer().maxStorage` should get encapsulated into a new method that is called "isWarehouseFull"